src/gitlabcache.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 Roger Meier <roger@bufferoverflow.ch>
+// SPDX-License-Identifier: MIT
+
+import Crypto from 'crypto';
+
+import {Logger} from '@verdaccio/types';
+import NodeCache from 'node-cache';
+
+export class GitlabCache {
+  private logger: Logger;
+  private ttl: number;
+  private storage: NodeCache;
+
+  public static get DEFAULT_TTL() {
+    return 300;
+  }
```

Comment:
Please excuse my question but why did you use a getter instead of `public readonly DEFAULT_TTL = 300;`?

Reply:
Because this was basically a quick hackjob, copied from authcache.ts.